flow/scripts/macro_place.tcl

```diff
@@ -41,14 +48,51 @@ proc find_macros {} {
 
 
 if {[find_macros] != ""} {
-  if {[info exists ::env(MACRO_PLACEMENT)]} {
-    source $::env(SCRIPTS_DIR)/read_macro_placement.tcl
-    puts "\[INFO\]\[FLOW-xxxx\] Using manual macro placement file $::env(MACRO_PLACEMENT)"
-    read_macro_placement $::env(MACRO_PLACEMENT)
+  if {[info exists ::env(RTLMP_FLOW)]} {
+    puts "RTLMP Flow enabled..."
+    partition_design -max_num_inst $env(RTLMP_MAX_INST) \
+                    -min_num_inst $env(RTLMP_MIN_INST) \
+                    -max_num_macro $env(RTLMP_MAX_MACRO) \
+                    -min_num_macro $env(RTLMP_MIN_MACRO) \
+                    -net_threshold 5 -virtual_weight 500 \
+                    -num_hop 5 -timing_weight 1 \
+                    -report_directory $env(RTLMP_RPT_DIR) \
+                    -report_file $env(RTLMP_RPT_FILE)
+ 
+    set flag [rtl_macro_placer -config_file $env(RTLMP_CONFIG_FILE) \
```

Comment:
Is `flag` used anywhere?

Reply:
Currently no. It was used to check if the placer completed successfully.

Reply:
Okay, please remove then.

Reply:
removed